funcs_utils.php

```diff
@@ -287,13 +287,14 @@ function current_branch_cms_major(
         return MetaData::HIGHEST_STABLE_CMS_MAJOR;
     }
 
+    $contents = '';
     if ($composerJson) {
         $contents = $composerJson;
     } elseif (check_file_exists('composer.json')) {
         $contents = read_file('composer.json');
     }
     $composerJson = json_decode($contents);
-    if (is_null($composerJson)) {
+    if (is_null($composerJson) && !is_misc()) {
```

Comment:
`$composerJson` will be null if there was an error.
We don't expect a composer.json file for most misc things - so if it's misc and has no file, we can ignore the error.
If this isn't misc, or if there is a file, we still want the error.
```suggestion
    if (is_null($composerJson) && !(is_misc() && !check_file_exists('composer.json'))) {
```
or 
```suggestion
    if (is_null($composerJson) && (check_file_exists('composer.json') || !is_misc())) {
```
or simply 
```suggestion
    if (is_null($composerJson) && check_file_exists('composer.json')) {
```
The last one takes away caring about whether it's misc or not - so if there's a file with errors we'll throw the error, and if there's no file we don't care about the error.

Reply:
Have gone with the bottom one, I have no idea why is_misc() is even in here